diff -uP lmbench3/src/Makefile lmbench3-patched/src/Makefile
--- lmbench3/src/Makefile	2005-08-22 19:19:54.000000000 -0500
+++ lmbench3-patched/src/Makefile	2015-02-17 11:20:24.000000000 -0600
@@ -4,6 +4,7 @@
 #
 # lmbench	[default] builds the benchmark suite for the current os/arch
 # results	builds, configures run parameters, and runs the benchmark
+# config	configures run parameters
 # rerun		reruns the benchmark using the same parameters as last time
 # scaling	reruns the benchmark using same parameters as last time,
 #		except it asks what scaling value to use
@@ -118,8 +119,10 @@
 	@env OS="${OS}" ../scripts/config-run
 	@env OS="${OS}" ../scripts/results
 
-rerun: lmbench
+config: lmbench
 	@if [ ! -f $(CONFIG) ]; then env OS="${OS}" ../scripts/config-run; fi
+
+rerun: lmbench config
 	@env OS="${OS}" ../scripts/results
 
 scaling: lmbench
@@ -165,41 +168,6 @@
 assembler:
 	@env CFLAGS=-O MAKE="$(MAKE)" MAKEFLAGS="$(MAKEFLAGS)" CC="${CC}" OS="${OS}" ../scripts/build asm
 
-bk.ver: ../SCCS/s.ChangeSet
-	rm -f bk.ver
-	-echo `bk prs -hr+ -d'$$if(:SYMBOL:){:SYMBOL: }:UTC:' ../ChangeSet;` > bk.ver
-	touch bk.ver
-
-dist: bk.ver
-	@if [ "X`cd ..; bk sfiles -c`" != "X" ]; then \
-		echo "modified files!"; \
-		false; \
-	 fi
-	@if [ "X`cd ..; bk pending`" != "X" ]; then \
-		echo "pending changes!"; \
-		false; \
-	 fi
-	cd ..; \
-		SRCDIR=`pwd`; \
-		DIR=`basename $${SRCDIR}`; \
-		VERSION=`cat src/bk.ver| awk '{print $$1;}' | sed -e 's/Version-//g'`; \
-		cd ..; \
-		bk clone $${DIR} /tmp/lmbench-$${VERSION}; \
-		cd /tmp/lmbench-$${VERSION}; \
-		bk sfiles | xargs touch; \
-		sleep 5; \
-		bk get -s; \
-		for d in doc results scripts src; do \
-			cd $$d; bk get -s; cd ..; \
-		done; \
-		bk sfiles -U -g | xargs touch; \
-		cd src; \
-		make bk.ver; \
-		cd /tmp; \
-		tar czf $${SRCDIR}/../lmbench-$${VERSION}.tgz \
-			lmbench-$${VERSION}; \
-		rm -rf /tmp/lmbench-$${VERSION};
-
 get $(SRCS):
 	-get -s $(SRCS)
 
@@ -228,9 +196,9 @@
 	install install-target dist get edit get-e clean clobber \
 	share depend testmake
 
-$O/lmbench : ../scripts/lmbench bk.ver
+$O/lmbench : ../scripts/lmbench
 	rm -f $O/lmbench
-	sed -e "s/<version>/`cat bk.ver`/g" < ../scripts/lmbench > $O/lmbench
+	sed -e "s/<version>/666/g" < ../scripts/lmbench > $O/lmbench
 	chmod +x $O/lmbench
 
 $O/lmbench.a: $(LIBOBJS)
